mesonbuild/environment.py

```diff
@@ -196,6 +196,8 @@ def get_llvm_tool_names(tool: str) -> T.List[str]:
     # unless it becomes a stable release.
     suffixes = [
         '', # base (no suffix)
+        '-19.1', '19.1',
+        '-19',  '19',
```

Comment:
We should have a final entry of `-20` for Debian's experimental snapshot, otherwise this looks good